provisioning/roles/kuma/files/vagrant/settings_local.py

```diff
@@ -108,3 +108,7 @@
     )
 
 SOCIALACCOUNT_PROVIDERS['persona']['AUDIENCE'] = 'https://developer-local.allizom.org'
+
+# Don't cache non-versioned static files in DEBUG mode
+if DEBUG:
+    WHITENOISE_MAX_AGE = 0
```

Comment:
I think that's the default, https://github.com/evansd/whitenoise/blob/0657e7aea986fa652bd04d6ae8468a0e1129d634/whitenoise/django.py#L58-L59 Unless you want to allow people to change it?